docs/arcaflow/index.md

```diff
@@ -36,21 +36,22 @@ You need to pick the right tool for the job. Sometimes, you need something simpl
     - It can run things on remote systems out of the box
     - It runs tasks as sequential (and, with some work, parallel) steps
     - You can pass data between steps
+    - Plugins and modules can be written in a variety of languages
+    - Tasks can run inside containers using custom connection plugins ([k8s example](https://docs.ansible.com/ansible/latest/collections/kubernetes/core/kubectl_connection.html))
 
     **What it does better than Arcaflow**
     
     - Well established and well known
     - Wide range of available plugins
     - Simple workflows are easy to write
-    - It runs comands over SSH by design
+    - It runs comands over SSH by default
+    - Users can define their own connection plugins
```

Comment:
I don't think this is a unique advantage of Ansible over Arcaflow for this section. Arcaflow also uses a modular system to define connections, known as deployers.

Reply:
moved to section of what they do similarly